### DISCUSSION TITLE
#36 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#11] [General]
Agenda:

- Welcome CryptoTechNight (9票全票通过)
- Community Updates
- Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by Thegaram** [comment]
**Crypto Tech Night 最近两周的更新**

3月24号举办了Crypto Tech Night #12 Web3: 属于Gen Z的社交网络时代。嘉宾：韩梦瑶（t²创始人&CEO），陈达（TokenPocket联合创始人），Yilan（CyberConnect Head of Content），0xAres（Web3科普博主）。观众：腾讯会议～100个人，Twitter~20个人。

CTN #13准备讨论The Merge，预计4月28号举办。

**Reply by Thegaram** [reply]
回放：https://www.bilibili.com/video/BV1Ci4y1k7HS

**Comment by 0x-stan** [comment]
# Dapp-Learning

近两周工作进展

## bounty

第一个task bounty ，关于 aave 和 uniswap 的 flashloan 使用范例。小伙伴已经完成主要内容，准备发放奖励。

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pull/643

## repo update

16个PR #664 ～ #688

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了3场分享会

- DODO算法解析与应用
  - by 雷达熊
  - https://www.youtube.com/watch?v=BhpBHT7691w
  - https://www.youtube.com/watch?v=Du96gGn64Lo
- Arbitrum源码解析与讨论
  - by arc
  - https://www.youtube.com/watch?v=hW5G6tL69YM
  - https://www.youtube.com/watch?v=pjW91KcaJig
- Web3Q and its Technologies
  - by Qi Zhou
  - https://www.youtube.com/watch?v=gygioYeQmKg

## 翻译小组

- 准备发布第 12、13集
- 翻译小组内部经验交流会议

**Comment by bobjiang** [comment]
- Gitcoin China
- Gitcoin grant 2.0
- https://gov.gitcoin.co/
- GitcoinDAO steward council
- ENSDAO ecosystem
- solidity learning

**Comment by winkybupt** [comment]
# 原语里弄
- 开始了四月共度活动，以了解密码学、计算机、黑客、互联网的发展史为主线，https://www.notion.so/6d08a94284ec4704a582878974da4bf2

**Comment by liushooter** [comment]
## Rebase

- 举办了一场 [online codingday](https://mp.weixin.qq.com/s/SF24bxehkNnNkp12nR_vYg)， 一共进行了6场分享
- 邀请 Ownership CEO 分享 Ceramic
- web3日报正常更新14期

**Comment by Riely007** [comment]
### 翻译小组

- 春季周边设计中

**Comment by Emilyxiannv** [comment]
### DAOSquare
产品分享会
一个建立在ceramic上的web3的产品， Orbis- decentralized social media app， Guest: Leon, a core member of Orbis. https://orbis.club/

**Comment by Jackieyewang** [comment]
1. 与seedao合作的Twitter space
2. 协助清华大学课程的宣传
